### PR TITLE
add PS column monitoring

### PR DESCRIPTION
**Pull request by mtosi**
https://github.com/cms-sw/cmssw/pull/14974 for 81x

### CODE REVIEW

**Comment by Martin-Grunewald** [comment]
please test

**Comment by Martin-Grunewald** [comment]
@dmitrijus, @vanbesien 
please sign or comment otherwise.

File: DQM/HLTEvF/plugins/PSMonitorClient.cc [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    edm::LogWarning("PSMonitorClient") << "no " << psColumnVSlumiPSet.name << " ME is available in " << psColumnVSlumiPSet.folder << std::endl;
+    return;
+  }
+  
+  TH2F* h = psColumnVSlumi->getTH2F();
+  size_t nbinsX = psColumnVSlumi->getNbinsX();
+  size_t nbinsY = psColumnVSlumi->getNbinsY();
+
+  for ( size_t ibinY=1; ibinY < nbinsY; ++ibinY )
+    std::cout << h->GetXaxis()->GetBinLabel(ibinY) << std::endl;
```
Comment (dmitrijus) [inline DQM/HLTEvF/plugins/PSMonitorClient.cc]: Please don't use thread unsafe std::cout.
If it's meant for debugging, put it under a flag, so it isn't triggered in production.

**Comment by dmitrijus** [comment]
Wrote a comment. Otherwise it looks fine.

**Comment by Martin-Grunewald** [comment]
please test

**Comment by mtosi** [comment]
@dmitrijus, @vanbesien,

**Comment by dmitrijus** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**